lib/monthly.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
+        SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+        private
+
+        IMPORTATION_VAT_RATES = [0.2, 0.1, 0.085, 0.055, 0.021, 0.0105].freeze
+
+        def initialize(declaration:, prefill:)
```

Comment:
The `initialize` method should take the company as the first argument. Please adjust the method signature to comply with the guidelines.